src/screens/FeedScreen/ContentPoll.js

```diff
@@ -170,14 +171,14 @@ const stylesComponent = (pollLength) =>
     feedDate: {
       fontFamily: fonts.inter[400],
       fontSize: normalizeFontSize(12),
-      color: COLORS.black,
+      color: colors.black,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color property has been updated from `COLORS.black` to `colors.black`. Make sure that `colors.black` is defined and is the intended color.
```diff
- color: COLORS.black
+ color: colors.black
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->